runtime/port/unix/j9sock.c

```diff
@@ -1225,14 +1227,14 @@ j9sock_getaddrinfo_name(struct J9PortLibrary *portLibrary, j9addrinfo_t handle,
 int32_t
 j9sock_gethostbyaddr(struct J9PortLibrary *portLibrary, char *addr, int32_t length, int32_t type, j9hostent_t handle)
 {
-#if defined(J9ZTPF)
+#if defined(J9ZTPF) || defined(OSX) /* Disable this on OSX to get to version */ 
```

Comment:
I don't understand the comment.

Reply:
This is just a marker to come back and look at the right answer for this function.  I've disabled to  "get to java -version" but it'll need to be revisited.

Reply:
ok - would be clearer if it  was "-version"